project-reports/2023/2023-Q4-Hyperledger-Aries.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: 2023 Q4 Hyperledger Aries
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+nav_exclude: true
+---
+
+# Project Health
+
+[Hyperledger Aries](https://www.hyperledger.org/projects/aries) continues at a
+healthy pace, seeing lots of activity in most of the Aries
+sub-communities, including [Aries Framework JavaScript], [Aries Bifold] (mobile
+wallet), [Aries Cloud Agent Python] and [Aries VCX]. Lots of progress being made
+on all of those efforts!
+
+While not a risk to the progress being made in the various open source Aries
+communities, the question of moving some of the Aries projects from the
+[Hyperledger Foundation] to the [Open Wallet Foundation] (OWF) was raised this quarter. A fork
+of a long dormant Aries project (the .NET Framework) has been created in the OWF. A more
+interesting situation is the transfer of the actively maintained [Aries
+Framework JavaScript] repository from Hyperledger to OWF that has been
+[proposed by some Aries Framework JavaScript maintainers]. It's an interesting topic, and
+we'll no doubt see more discussion and action related to Aries and the OWF in coming months.
+
+[Aries Framework JavaScript]: https://github.com/hyperledger/aries-framework-javascript
+[Aries Bifold]: https://github.com/hyperledger/aries-mobile-agent-react-native
+[Aries Cloud Agent Python]: https://github.com/hyperledger/aries-cloudagent-python
+[Aries VCX]: https://github.com/hyperledger/aries-vcx
+[Aries Framework Go]: https://github.com/hyperledger/aries-framework-go
+[proposed by some Aries Framework JavaScript maintainers]: https://github.com/hyperledger/aries-framework-javascript/discussions/1586
+[Open Wallet Foundation]: https://openwallet.foundation/
+[Hyperledger Foundation]: https://hyperledger.org
+
+# Questions/Issues for the TOC
+
+None.
+
+# Releases
+
+The following Aries releases occurred in the last quarter:
+
+- [Aries Cloud Agent Python] -- 0.10.0, 0.10.1, 0.10.2, 0.10.3
+- [Aries Framework JavaScript] -- v0.4.0, v0.4.1alpha.5-34, v0.4.1, v0.4.2, v0.5.0alpha57-58
+- [Aries Askar] -- v0.3.0-dev1
+- [Aries VCX Releases] -- v0.58.0-v0.61.0
```

Comment:
Nitpick: the link doesn't display here, this should be `[Aries VCX]`

Reply:
Thanks — fixed.